src/App.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./App.css";
+import "./styles/variables.css";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Navbar from "./components/Header/Navbar";
+import Footer from "./components/Footer/Footer";
+import Home from "./pages/Home";
+import Menu from "./pages/Menu";
+import Reservation from "./pages/Reservation";
+
+function App() {
+  return (
+    <Router>
+      <div className="App">
+        <header>
+          <Navbar />
+        </header>
+
+        <main>
+          <Routes>
+            <Route exact path="/" element={<Home />} />
+            <Route path="/menu" element={<Menu />} />
+            <Route path="/reservation" element={<Reservation />} />
+          </Routes>
+        </main>
+
+        <footer>
+          <Footer />
+        </footer>
```

Comment:
Instead of doing this, just use the `footer` HTML tag in your `<Footer />` component. 
Then, you don't need to repeat it twice here. So it should just say:
```
<Footer />
```